app/javascript/components/pages/DeckShow.js

```diff
@@ -10,22 +14,75 @@ export class DeckShow extends Component {
 		}
 	}
 
-	onClick = () => {
-		if (this.state.class === 'term') {
-			this.setState({ class: 'definition' })
+	// Toggle between "Term" and "Definition"
+	onToggle = () => {
+		if (this.state.isTerm === true) {
+			this.setState({ isTerm: false })
 		} else {
-			this.setState({ class: 'term' })
+			this.setState({ isTerm: true })
+		}
+	}
+
+	prevClick = () => {
+		const { cards, index } = this.state
+		let newIndex = index - 1
+		let lastIndex = cards.length - 1
+		if (index === 0) {
+			this.setState({ index: lastIndex })
+		} else if (index > 0) {
+			this.setState({ index: newIndex })
+		}
+
+		// if index = 0 then setState index: this.state.cards.length
+
+		console.log('newIndex:', this.state.index)
+	}
+
+	nextClick = () => {
+		const { cards, index } = this.state
+		let newIndex = index + 1
+		if (index < cards.length - 1) {
+			console.log('cards length', cards.length)
+			console.log('index', index)
+			this.setState({ index: newIndex })
+		} else if (index === cards.length - 1) {
+			this.setState({ index: 0 })
 		}
+
+		console.log('newIndex:', this.state.index)
 	}
 
 	render() {
 		const { cards, index, isTerm } = this.state
 		console.log('state on show page:', this.state)
 		return (
 			<Fragment>
-				<div className='deck-title-card'>
-					<h4>{cards[index].term}</h4>
-					<p>{cards[index].definition}</p>
+				<div className='show-card-container show-page'>
+					<div className='deck-show-card' onClick={this.onToggle}>
```

Comment:
```suggestion
					<div className='deck-show-card' onClick={() => this.onToggle(isTerm)}>
```

Reply:
If you choose to make the `onToggle` change, you would need to update this passing of the function because it now requires a parameter